pom.xml

```diff
@@ -400,6 +400,16 @@
       <artifactId>checker-qual</artifactId>
       <version>${checkerframework.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.maven.doxia</groupId>
+      <artifactId>doxia-core</artifactId>
+      <version>1.12.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.maven.doxia</groupId>
+      <artifactId>doxia-module-xdoc</artifactId>
+      <version>1.12.0</version>
+    </dependency>
   </dependencies>
```

Comment:
Snyk is failing. I believe it is one of those but I do not have access to see.

Reply:
commons-codec:commons-codec
Information Exposure
VULNERABILITY
Introduced through
org.apache.maven.doxia:doxia-core@1.12.0
Fixed in
commons-codec:commons-codec@1.13
Exploit maturity
NO KNOWN EXPLOIT
Show less detail 
Detailed paths
Introduced through: com.puppycrawl.tools:checkstyle@10.12.2-SNAPSHOT › org.apache.maven.doxia:doxia-core@1.12.0 › org.apache.httpcomponents:httpclient@4.5.13 › commons-codec:commons-codec@1.11
Security information
Factors contributing to the scoring:
Snyk: [CVSS 3.7](https://security.snyk.io/vuln/SNYK-JAVA-COMMONSCODEC-561518) - Low Severity

NVD: NVD only publishes analysis of vulnerabilities which are assigned a CVE ID. This vulnerability currently does not have an assigned CVE ID.
[Why are the scores different? Learn how Snyk evaluates vulnerability scores](https://docs.snyk.io/features/fixing-and-prioritizing-issues/issue-management/severity-levels#understanding-snyks-vulnerability-analysis)
Overview
[commons-codec:commons-codec](https://commons.apache.org/proper/commons-codec) is a package that contains simple encoder and decoders for various formats such as Base64 and Hexadecimal.

Affected versions of this package are vulnerable to Information Exposure. When there is no byte array value that can be encoded into a string the Base32 implementation does not reject it, and instead decodes it into an arbitrary value which can be re-encoded again using the same implementation. This allows for information exposure exploits such as tunneling additional information via seemingly valid base 32 strings.

Remediation
Upgrade commons-codec:commons-codec to version 1.13 or higher.